e2e/e2e_test.go

```diff
@@ -0,0 +1,166 @@
+package e2e_test
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"testing"
+
+	"dagger.io/dagger"
+	"github.com/spf13/cobra"
+)
+
+var runTests = &cobra.Command{
+	Use:   "test",
+	Short: "run k6 tests",
+	Run:   e2eTests,
+}
+
+var errorHandling = &cobra.Command{
+	Use:   "error handling",
+	Short: "check if k6 tests have failing tests",
+	RunE: func(cmd *cobra.Command, args []string) error {
```

Comment:
Instead of making another command, you could add `PostRunE` to `runTests`

Reply:
I didn't realize you could do that! Nice 💯